torchvision/prototype/transforms/_deprecated.py

```diff
@@ -52,7 +51,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 class ToPILImage(Transform):
 
     # Updated transformed types for ToPILImage
-    _transformed_types = (torch.Tensor, features._Feature, PIL.Image.Image, np.ndarray)
+    _transformed_types = (is_simple_tensor, features._Feature, PIL.Image.Image, np.ndarray)
```

Comment:
Only `features.Image` should be allowed here instead of `features._Feature`. Everything else will fail.

Reply:
@pmeier Victor is doing changes only on the is_simple_tensor to replace existing torch.Tensor. I'll be following up with a broader clean up here.